### ISSUE TITLE
Support for postgresql 16

### ISSUE THREAD

**Issue by e12e** [UpCloudLtd/terraform-provider-upcloud#570]
Same as #365 - but for v16:

```console
tofu plan -out do.plan
(...)

Planning failed. OpenTofu encountered an error while generating this plan.

╷
│ Error: expected version to be one of ["12" "13" "14" "15"], got 16
│ 
│   with upcloud_managed_database_postgresql.wildebeast,
│   on wildebeast.tf line 20, in resource "upcloud_managed_database_postgresql" "wildebeast":
│   20:     version       = "16"
│ 
╵
```
tofu version output:
```console
OpenTofu v1.7.2
on darwin_arm64
+ provider registry.opentofu.org/hashicorp/null v3.2.2
+ provider registry.opentofu.org/hashicorp/random v3.6.2
+ provider registry.opentofu.org/upcloudltd/upcloud v5.6.1
```

**Comment by kangasta** [comment]
Thank you for raising this up! We will create a new release with the support added.

**Comment by kangasta** [comment]
Support for Postgres 16 should now be available in [v5.7.0](https://github.com/UpCloudLtd/terraform-provider-upcloud/releases/tag/v5.7.0) 🚢
